### DISCUSSION TITLE
Change [Bugs] title in releases to BugFixes or similair

### DISCUSSION THREAD

**Discussion by migs35323** [longhorn/longhorn#6390] [Q&A] [answered]
In longhorn releases in the "changelog" there are several markdown tags like improvement, resilience, stability etc
however there is the "BUGS" which at first glance i always have to check if these are bugs still present in this version, or bugs that are new, or bugs fixed
i believe that new people get a bit confused when they seen it, i had used this project for some time and i always have to check twice that the [bugs] section are actually bugfixes and not bugs found in that release. not sure if it was in longhorn or other project but i remember once seing "bugs" and "bufixes" in a release document

**Comment by mantissahz** [comment]
cc @innobead

**Reply by innobead** [reply, marked as answer]
@migs35323 
Good feedback. We will improve it in the following release notes.
